.github/workflows/master.yml

```diff
@@ -193,6 +196,8 @@ jobs:
           # 4.46.0 is broken because of https://github.com/huggingface/transformers/issues/34370
           # TODO: remove this once above issue is resolved
           pip install 'transformers!=4.46.0'
+          # transformers doesn't support Keras 3 yet. tf-keras needs to be installed as a workaround.
```

Comment:
Is this a limitation on python 3.9?

Reply:
Yes, Keras 3 only supports Python >= 3.9.